checks/cloud/aws/iam/no_policy_wildcards.rego

```diff
@@ -29,3 +29,5 @@
 #             provider: aws
 #   examples: checks/cloud/aws/iam/no_policy_wildcards.yaml
 package builtin.aws.iam.aws0057
+
+# TODO(simar7): This checks seems to have lost its flesh during the transition.
```

Comment:
@nikpivkin I found this check got lost in the transition. We should add it back.

Reply:
Ah I just realized this is a deprecated check 😄